app/survey_config/survey_config.py

```diff
@@ -41,3 +39,22 @@ class SurveyConfig:
     survey_title: Optional[LazyString] = None
     design_system_theme: Optional[str] = None
     data_layer: Iterable[Union[Mapping]] = field(default_factory=list, compare=False)
+    sign_out_button_text: str = lazy_gettext("Save and sign out")
+    contact_us_url: str = field(init=False)
+    cookie_settings_url: str = field(init=False)
+    privacy_and_data_protection_url: str = field(init=False)
+
+    def __post_init__(self):
+        self.contact_us_url: str = f"{self.base_url}/contact-us/"
+        self.cookie_settings_url: str = f"{self.base_url}/cookies/"
+        self.privacy_and_data_protection_url: str = (
+            f"{self.base_url}/privacy-and-data-protection/"
```

Comment:
:thinking:

Reply:
Due to https://surveydatacollection.slack.com/archives/CLR17KQKU/p1642770634085600